pex/bin/pex.py

```diff
@@ -585,13 +573,16 @@ def get_interpreter(python_interpreter, interpreter_cache_dir, repos, use_wheel)
 
 def build_pex(args, options, resolver_option_builder):
   with TRACER.timed('Resolving interpreters', V=2):
-    interpreters = [
-      get_interpreter(interpreter,
-                      options.interpreter_cache_dir,
-                      options.repos,
-                      options.use_wheel)
-      for interpreter in options.python or [None]
-    ]
+    def to_python_interpreter(interpreter):
+      if os.path.exists(interpreter):
+        return PythonInterpreter.from_binary(interpreter)
+      else:
+        pi = PythonInterpreter.from_env(interpreter)
```

Comment:
I immediately thought of 3.14 haha. I would have wanted to name this variable `interpreter`, but see its already taken. Which raises the question, how is this interpreter different than the parameter? Is there a name we could use like `interpreter_from_env` that reflects the semantic difference?

Reply:
Explicitified.